simulation/test_torrent_status.cpp

```diff
@@ -76,35 +99,82 @@ TORRENT_TEST(status_timers)
 			// to stay within that range
 			if (ticks > 20 * 60 * 60)
 			{
-				// unfortunantly we can't test this many ticks
-				TEST_ERROR("This line should be reached but doesn't");
 				return true;
 			}
 
-			if (ticks == 0){
-				// torrent get ready for seeding on first tick, which means +1s
-				startseed_time = lt::clock_type::now();
+			if(tick_is_in_active_range){
+				// 1 second per tick
+				active_time++;
 			}
 
+			// resume on a tick which is far away from next tick check intervall
+			// because with very low upload limit it takes some time to unchock
+			switch(ticks)
```

Comment:
Or do you mean the comment
```
// resume on a tick which is far away from next tick check intervall
// because with very low upload limit it takes some time to unchock
```
?

Without limit I got upload aktivity on tick 1, but with rate limit the first upload activity is on tick 11. After the first start or resume unchock it get better and I get upload activity every 5 ticks.